src/components/GradeBookContentContainer.tsx

```diff
@@ -88,6 +116,19 @@ function GradeBookContentContainer() {
 						<CourseItemTitle>Grade</CourseItemTitle>
 						<CourseItemTitle>Due Date</CourseItemTitle>
 					</tr>
+					{course.items.map((item, index) => {
+						console.log({ item, index });
+						function callback(newItem: CourseItem) {
+							updateCourseItem(newItem, index);
+						}
+						return (
+							<GradeItemAccordion
+								key={index}
+								item={item}
+								updateItem={callback}
+							/>
+						);
```

Comment:
Lets update this so the callback is just defined inline.
```suggestion
						return (
							<GradeItemAccordion
								key={index}
								item={item}
								updateItem={(newItem) => updateCourseItem(newItem, index)}
							/>
						);
```